src/v/renderers/BppMode.java

```diff
@@ -1,18 +1,50 @@
+/*
+ * Copyright (C) 2011-2013 velktron
```

Comment:
Please remove this line. I have contacted Maes and he has to confirm how he wants his copyright displayed before we put his name here.